packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -511,7 +510,7 @@ $Calendar-dayPicker-margin: 10px;
 
 .wrap span:focus,
 .wrap div:focus {
-  outline: 1px solid $ms-color-themePrimary;
+  outline: $ms-color-themePrimary;
```

Comment:
We have a focus border mixin that should help https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/common/_focusBorder.scss